soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2082,6 +2083,8 @@ void Player_ProcessControlStick(PlayState* play, Player* this) {
         direction = (u16)((s16)(sControlStickWorldYaw - this->actor.shape.rot.y) + 0x2000) >> 14;
     }
 
+    Mouse_UpdateQuickspinCount();
+
```

Comment:
Can we get these additions isnide z_player.c inside either their own hooks or their own VB_Shoulds? Ideally in places where it might be useful to other enhancements down the line as well. I'm not really familiar with this part of the code so I wouldn't know what the best path forward here would be, but there being 5 calls within vanilla code doesn't sit well with me. 

I'd ask for the same in z_camera.c but I see a lot of that code is already touched by freecam etc so I won't block the PR because of it.